Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -245,8 +245,11 @@ def build_iterator(self, **kwargs):
     def custom_fields_creator(self, attributes: dict):
         created_custom_fields = {}
         for attribute in attributes.keys():
-            if attribute in self.custom_fields_mapping.keys():
-                created_custom_fields[self.custom_fields_mapping[attribute]] = attributes[attribute]
+            if attribute in self.custom_fields_mapping.keys() or attribute == 'tags':
```

Comment:
lets make one list that contain the keys and `'tags'` and then iterate it
`'tags'` should be const